docs/troubleshooting.md

```diff
@@ -0,0 +1,141 @@
+ NAME            AGE
+ my-lvmcluster   65m
+ ```
+
+If an LVMCluster resource is not found, you can create one based on your requirements. [Here](../config/samples/lvm_v1alpha1_lvmcluster.yaml) is a sample one you can use.
+
+```bash
+$ oc create -n openshift-storage -f https://github.com/openshift/lvm-operator/raw/main/config/samples/lvm_v1alpha1_lvmcluster.yaml
+```
+ 
+If an `LVMCluster` already exists, check if all the pods from LVMS are in `Running` state on `openshift-storage` namespace:
+ 
+ ```bash 
+ $ oc get pods -n openshift-storage
+ NAME                                  READY   STATUS    RESTARTS      AGE
+ lvms-operator-7b9fb858cb-6nsml        3/3     Running   0             70m
+ topolvm-controller-5dd9cf78b5-7wwr2   5/5     Running   0             66m
+ topolvm-node-dr26h                    4/4     Running   0             66m
+ vg-manager-r6zdv                      1/1     Running   0             66m
+ ```
+ 
+There should be one running instance of `lvms-operator` and `vg-manager`, and multiple instances of `topolvm-controller` and `topolvm-node` depending on number of nodes.
+
+#### `topolvm-node` is stuck in `Init:0/1`
+
+This indicates a failure in locating an available disk for LVMS utilization. To investigate further and obtain relevant information, review the logs of the `vg-manager` pod.
+
+```bash
+$ oc logs vg-manager -n openshift-storage
+```
+
+### Disk failure
+
+If you encounter a failure message such as "failed to check volume existence" while inspecting the events associated with the PVC, it indicates a potential issue related to the underlying volume or disk. This failure message suggests that there might be a problem with the availability or accessibility of the specified volume. Further investigation is recommended to identify the exact cause and resolve the underlying issue.
+
+```bash
+$ oc describe pvc lvms-test  
+Events:
+Type     Reason              Age               From                         Message
+----     ------              ----              ----                         -------
+Warning  ProvisioningFailed  4s (x2 over 17s)  persistentvolume-controller  failed to provision volume with StorageClass "lvms-vg1": rpc error: code = Internal desc = failed to check volume existence
+```
+
+To investigate the issue further, you can establish a direct connection to the host where the problem is occurring. From there, you can proceed with creating a new file to gather additional information and gain insights into the underlying problem related to the disk. After resolving the underlying disk problem, if the recurring issue persists despite the resolution, it may be necessary to perform a [forced cleanup procedure](#forced-cleanup) for LVMS. After completing the cleanup process, re-create the LVMCluster. By re-creating the LVMCluster, all associated objects and resources are recreated, providing a clean starting point for the LVMS deployment. This helps to ensure a reliable and consistent environment.
```

Comment:
```suggestion
To investigate the issue further, establish a direct connection to the host where the problem is occurring. From there, create a new file to gather additional information and gain insights into the underlying problem related to the disk. After resolving the underlying disk problem, if the recurring issue persists despite the resolution, it may be necessary to perform a [forced cleanup procedure](#forced-cleanup) for LVMS. After completing the cleanup process, re-create the `LVMCluster` CR. When you re-create the `LVMCluster` CR, all associated objects and resources are recreated, providing a clean starting point for the LVMS deployment. This helps to ensure a reliable and consistent environment.
```